app/common/themes/GristDark.ts

```diff
@@ -25,6 +25,7 @@ export const GristDark: ThemeTokens = {
   primaryMuted: '#1da270',
   primaryDim: '#157a54',
   primaryEmphasis: '#13d78d',
+  primaryTransparent: 'rgba(23, 179, 120, 0.5)',
```

Comment:
This is the old notation. The modern one would be:
```suggestion
  primaryTransparent: 'rgb(23 179 120 / 0.5)',
```

Reply:
You are right! I just stuck to the notation already used in the project, will certainly change this later as a whole ;) Thanks!